### PR TITLE
#42 List task entries

### LINKED ISSUE #42: List task entries.

### PR DESCRIPTION
**Pull request by brayvasq**
Closes #42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -518,6 +519,79 @@ const result = await client.tasks.getTask(1, callback);
...
+
+#### List Task Entries
+
+This will return all entries that are related to a specific task and meet the provided parameters. You can send some params to filter the response, those params are the following:
+
+- [Required] taskId, related parent task.
+- [Required] startDate. The format is: 'YYYY-MM-DD'.
+- [Required] endDate. The format is: 'YYYY-MM-DD'.
+- [Optional] userId, will be ignored if the user is not an administrator.
+- [Optional] userId, will be ignored if the user is not an administrator.
```
Comment (MrJuanGaviriaK) [inline README.md:531]: this line is repeated.

File: src/v2/resources/entries.js [hunk 2]
```diff
@@ -29,20 +30,19 @@ class Entries extends BaseResource {
...
+      startDate,
+      endDate,
+      userId,
+      projectId,
+      taskId,
+      billable,
+      billed,
+    }, {
+      baseURL: this.baseURL,
+      module: 'entries',
```
Comment (MrJuanGaviriaK) [inline src/v2/resources/entries.js:43]: Do you need to pass this module if the default module is entries? ¿What do you think?

**Review by MrJuanGaviriaK — CHANGES_REQUESTED** [review]
Hi @brayvasq nice job 👏 , I left a few comments.

File: src/v2/resources/entries.js (hunk 2 above)
Comment (brayvasq) [reply src/v2/resources/entries.js]: I do not think so. It could be inferred from the class. I will remove it.

**Comment by brayvasq** [comment]
> Hi @brayvasq nice job clap , I left a few comments.

@MrJuanGaviriaK Done! Thanks for the comments :+1:

**Review by MrJuanGaviriaK — APPROVED** [review]

**Pull request merged**